web3_multi_provider/metrics.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+PROMETHEUS_PREFIX = os.getenv("PROMETHEUS_PREFIX", "")
+
+CHAIN_ID_TO_NAME = {
```

Comment:
I propose to not use such hardcoded names.
Create some config structure and developer will be responsible to provide network from app

```python
class MultiWeb3Config:
    use_metrcis: bool
    network: str  # Network name
```